Makefile

```diff
@@ -82,12 +82,21 @@ ifneq ($(OVERRIDE_IMAGE_NAME),)
 endif
 	rm -rf $(TEMP_DIR)
 
-push: ./manifest-tool gcr-login $(addprefix sub-push-,$(ALL_ARCHITECTURES))
-	./manifest-tool push from-args --platforms $(ML_PLATFORMS) --template $(PREFIX)/heapster-ARCH:$(VERSION) --target $(PREFIX)/heapster:$(VERSION)
+do-push:
+	docker push $(PREFIX)/heapster-$(ARCH):$(VERSION)
+ifeq ($(ARCH),amd64)
```

Comment:
IIUC you push amd64 image as `heapster:<version>` without explicit specifying architecture. Why do you need an architecture in Heapster manifest then?

Also why don't you have the same thing for grafana/influx?